sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/TransmissionStateManager.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Internals
+{
+    internal class TransmissionStateManager
+    {
+        private int _consecutiveErrors;
+
+        private const long MaxDelayInMilliseconds = 3600000;
+
+        internal const long MinDelayInMilliseconds = 10000;
+
+        private readonly Random _random = new();
```

Comment:
Why these fields are readonly if not initialized inside constructor?

Reply:
Without that as well, we need it to be readonly